log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
shouldn't be this file in resources? to get it on classpath

Reply:
yep, just for experimenting, thx